app/Exceptions/ResumeBootcampNotFoundException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Exceptions;
+
+use Exception;
+use Throwable;
+
+class ResumeBootcampNotFoundException extends Exception
+{
+    public const MESSAGE = 'L\'estudiant amb ID: %s no té informat cap Bootcamp al seu currículum';
+
+    public function __construct(string $studentId, Throwable $previous = null)
+    {
+        $message = sprintf(self::MESSAGE, $studentId);
+        parent::__construct($message, 404, $previous);
+    }
```

Comment:
In accordance with Jordi's review of my latest PR, I suggest creating the constructor like this:
~~~
public function __construct(string $studentId, $code = 404, Throwable $previous = null)
{
        $message = sprintf(self::MESSAGE, $studentId);
        parent::__construct($message, $code, $previous);
}
~~~